frontend/src/app/modules/boards/board/add-list-modal/add-list-modal.html

```diff
@@ -16,15 +16,27 @@
         <div class="form--field-container">
           <div class="form--select-container">
             <label class="form--label" [textContent]="actionService.label"></label>
-            <ndc-dynamic [ndcDynamicComponent]="autocompleterComponent()"
-                         [ndcDynamicInputs]="{ availableValues: availableValues,
-                                               typeahead: requests.input$,
-                                               appendTo: 'body',
-                                               model: '',
-                                               classes: 'new-list--action-select',
-                                               finishedLoading: true}"
-                         [ndcDynamicOutputs]="referenceOutputs">
-            </ndc-dynamic>
+            <op-autocompleter #select
+            classes="new-list--action-select"
+            [appendTo]="'body'"
```

Comment:
There is no need to use binding to pass down strings, it can be done like this: `appendTo="body"`